packages/components/organisms/f-offers/package.json

```diff
@@ -57,8 +57,8 @@
   },
   "devDependencies": {
     "@braze/web-sdk": "3.3.0",
+    "@justeat/f-content-cards": "6.0.0-beta.2",
```

Comment:
Aren't those supposed to be part of the `dependencies`?

Reply:
our current convention is devDependencies for imported components. Not sure I agree but thats the current convention.

Reply:
If they're in `dependencies` then we're essentially saying that the consuming app needs requires them. 

Would a web project need to download `f-content-cards` in order to use the `f-offers` npm package? Or does it already come pre-bundled? 

I'm guessing the latter. If so, it should go in `devDependencies` so that the consuming app doesn't download it.

Reply:
I can see from the generated report in `f-offers` that `f-content-cards` are pre-bundled in the resulting npm package.

![Screenshot 2021-09-28 at 12 26 17](https://user-images.githubusercontent.com/89059/135078228-e02a52b3-1703-43cf-ad2a-0016ddc59684.png)